analyzer/consensus/consensus.go

```diff
@@ -122,9 +122,15 @@ func (m *processor) FinalizeFastSync(ctx context.Context, lastFastSyncHeight int
 	if r.GenesisHeight == firstSlowSyncHeight {
 		m.logger.Info("fetching genesis document before starting with the first block of a chain", "chain_context", r.ChainContext, "genesis_height", r.GenesisHeight)
 		genesisDoc, err = m.source.GenesisDocument(ctx, r.ChainContext)
+		if genesisDoc != nil {
+			m.dumpGenesisJSON(ctx, genesisDoc, r.ArchiveName)
```

Comment:
```suggestion
			m.debugDumpGenesisJSON(ctx, genesisDoc, r.ArchiveName)
```

So it's more clear that this is more of a debugging helper.